src/qt/walletcontroller.cpp

```diff
@@ -124,3 +119,26 @@ void WalletController::removeAndDeleteWallet(WalletModel* wallet_model)
     // CWallet shared pointer.
     delete wallet_model;
 }
+
+
+OpenWalletActivity::OpenWalletActivity(WalletController* wallet_controller, const std::string& name)
+    : m_wallet_controller(wallet_controller)
+    , m_name(name)
+{}
+
+void OpenWalletActivity::open()
+{
+    std::string error, warning;
+    std::unique_ptr<interfaces::Wallet> wallet = m_wallet_controller->m_node.loadWallet(m_name, error, warning);
+    if (!warning.empty()) {
+        Q_EMIT message(QMessageBox::Warning, QString::fromStdString(warning));
+    }
+    if (wallet) {
+        printf("wallet loaeded!\n");
```

Comment:
Nit: don't forget to remove (or at least fix typo)

Reply:
Done.